src/Workspaces/Core/Portable/Remote/RemoteFeatureOptions.cs

```diff
@@ -11,49 +10,10 @@ internal static class RemoteFeatureOptions
-        /// user visible switch to let people opt-in/out of this behavior.
-        /// </summary>
-        public static readonly Option<bool> OutOfProcessAllowed = new Option<bool>(
-            nameof(RemoteFeatureOptions), nameof(OutOfProcessAllowed), defaultValue: false,
-            storageLocations: new LocalUserProfileStorageLocation(LocalRegistryPath + nameof(OutOfProcessAllowed)));
-
-        // Individual feature switches.  Not exposed to the user.  Supplied as an escape hatch for
-        // features if necessary.  If all features use OOP then no indices will need to be built
-        // within VS.  However, if any features need to run in VS, then we have to build our indices
-        // in VS as well.
-
-        public static readonly Option<bool> AddImportEnabled = new Option<bool>(
-            nameof(RemoteFeatureOptions), nameof(AddImportEnabled), defaultValue: true,
-            storageLocations: new LocalUserProfileStorageLocation(LocalRegistryPath + nameof(AddImportEnabled)));
-
-        public static readonly Option<bool> DocumentHighlightingEnabled = new Option<bool>(
-            nameof(RemoteFeatureOptions), nameof(DocumentHighlightingEnabled), defaultValue: true,
-            storageLocations: new LocalUserProfileStorageLocation(LocalRegistryPath + nameof(DocumentHighlightingEnabled)));
-
-        public static readonly Option<bool> NavigateToEnabled = new Option<bool>(
-            nameof(RemoteFeatureOptions), nameof(NavigateToEnabled), defaultValue: true,
-            storageLocations: new LocalUserProfileStorageLocation(LocalRegistryPath + nameof(NavigateToEnabled)));
-
-        public static readonly Option<bool> SymbolSearchEnabled = new Option<bool>(
-            nameof(RemoteFeatureOptions), nameof(SymbolSearchEnabled), defaultValue: true,
-            storageLocations: new LocalUserProfileStorageLocation(LocalRegistryPath + nameof(SymbolSearchEnabled)));
-
-        public static readonly Option<bool> SymbolFinderEnabled = new Option<bool>(
-            nameof(RemoteFeatureOptions), nameof(SymbolFinderEnabled), defaultValue: true,
-            storageLocations: new LocalUserProfileStorageLocation(LocalRegistryPath + nameof(SymbolFinderEnabled)));
-
         public static readonly Option<bool> DiagnosticsEnabled = new Option<bool>(
             nameof(RemoteFeatureOptions), nameof(DiagnosticsEnabled), defaultValue: true,
             storageLocations: new LocalUserProfileStorageLocation(LocalRegistryPath + nameof(DiagnosticsEnabled)));
 
-        private static ImmutableArray<Option<bool>> AllFeatureOptions { get; } =
-            ImmutableArray.Create(AddImportEnabled, DocumentHighlightingEnabled, NavigateToEnabled, SymbolSearchEnabled, SymbolFinderEnabled);
-
-        public static bool AnyFeatureRunsInProcess(Workspace workspace)
-            => AllFeatureOptions.Any(o => !workspace.IsOutOfProcessEnabled(o));
-
         public static bool ShouldComputeIndex(Workspace workspace)
```

Comment:
not sure why we still need this.

Reply:
This is still necessary because computing the index should only happen in certain workspaces.